docs/eslint.md

```diff
@@ -21,7 +21,7 @@ yarn add --dev prettier eslint-plugin-prettier
 }
 ```
 
-We also recommend that you use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to disable all the existing formatting rules. It's a one liner that can be added on-top of any existing ESLint configuration.
+We also recommend that you use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to disable all the existing formatting rules. It's a one liner that can be added on-top of any existing ESLint configuration. (You still need to install the plugin and add the `plugins` and `rules` directives.)
```

Comment:
What `rules` directives? I don't understand what's confusing before

Reply:
The documentation first tells you to use `eslint-plugin-prettier` with plugin and rules directives.

Immediately afterwards, it suggests you also use `eslint-config-prettier` and shows an example `.eslintrc.json` file, formatted as a full file.  But, unless I'm confused, you would *never* actually make an `.eslintrc.json` file that looks like the example for `eslint-config-prettier`: you would *always* be using it as an addition to `eslint-plugin-prettier`.

When I read the docs, I thought "ah, ok, `eslint-config-prettier` is a module that lets you pre-configure a bunch of things that are useful for prettier like turning off irrelevant rules, and presumably it also turns on the prettier rule so that you can easily load all of prettier with a single line in your file". But that's not true (and maybe it can't be true, I have no idea if `extends` things that pull in `plugins`). So I just thought it would be nice to be more explicit.

Alternatively, the example `.eslintrc.json` for `eslint-config-prettier` in the docs here could just include the plugins/rules instead of me saying this in the text.  But it seems misleading to show something that looks like a full `.eslintrc.json` that doesn't actually enable `prettier`.

Reply:
Ah gotcha :+1: Thanks for explaining!

Now that I understand what you mean, I don't think this diff does that. I think it would be best to just be more explicit that the `config` and `plugin` (specially that they do different things). Also, I believe we should mention the config first, then the plugin and actually explain what the plugin does (I don't think it does that at the moment).

Something in the likes of:

### Integratin with ESLint

If you use ESLint, use `eslint-config-prettier` to [explain here what it does].

You can also use ESLint to run Prettier for you by using `eslint-plugin-prettier`

Reply:
I actually don't understand in what context you wouldn't want to use both?

Reply:
One example would be to run eslint after running prettier, i.e. `prettier-eslint` or a custom pipeline that can run other tools in sequence.

Reply:
OK, I basically rewrote the whole file. How's it look?